### ISSUE TITLE
Accessibility Audit: 'View On Map' link on Listing Detail Page has misleading label

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#947]
**User Story:** 

As a….person using a screen reader
I want…links on the page to be accompanied with a description of the action I would be taking when I follow that link
So that…I know where I would land if I followed the link

**Additional Detail:**

- Design asset required? No
- Go to a [Listing Detail](https://homeconnect.detroitmi.gov/listing/dfce011e-e514-43f1-bced-6cf5cd0386ac/preserve_on_ash_i_3309_14_th_street_detroit_mi) page. The 'View on Map' link underneath the listing address and the "Get directions" link under the map at the bottom of the page have an aria-label attribute of 'Opens in new window'.  This aria overrides the link label, so we need to update the aria to say something that is more action oriented to avoid a confusing user experience. The label should be 'View on Map' for both links. 

UPDATE: the "View on Map" link doesn't require a label since the link text itself provides the same description. The AC below will still be meet with no aria-label on that link.

**Technical Notes:** 
- Change to UI component required?

**Acceptance criteria:** 
- The "view on map link" under the address and the "get directions" link under the map view on the bottom of the page should be read as "view on map" on a screen reader.

**QA review instructions:**
TBD

**Comment by sarahlazarich** [comment]
Note from audit sheet: Link is unlabeled, should say "Get Directions" which is the same text that appears on the screen.

**Comment by sarahlazarich** [comment]
The "Get Directions" link now reads "view on map" - lgtm!
